resources/views/campaigns/members/_invites.blade.php

```diff
@@ -109,4 +108,29 @@
             <div class="modal-content rounded-2xl bg-base-100" id="small-modal-content"></div>
         </div>
     </div>
+
+    <div class="modal fade" id="datagrid-bulk-delete" tabindex="-1" role="dialog" aria-labelledby="clickConfirmLabel">
```

Comment:
We already have a modal for deleting confirmation embedded in every single page. No need to create a new one.